webpack/scenes/RedHatRepositories/index.js

```diff
@@ -7,16 +7,21 @@ import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import { Grid, Row, Col } from 'react-bootstrap';
 import { LoadingState } from '../../move_to_pf/LoadingState';
-
 import { createEnabledRepoParams, loadEnabledRepos } from '../../redux/actions/RedHatRepositories/enabled';
 import { loadRepositorySets, updateRecommendedRepositorySets } from '../../redux/actions/RedHatRepositories/sets';
 import SearchBar from './components/SearchBar';
 import RecommendedRepositorySetsToggler from './components/RecommendedRepositorySetsToggler';
 import { getSetsComponent, getEnabledComponent } from './helpers';
+import { orgId } from '../../services/api';
 
 class RedHatRepositoriesPage extends Component {
   componentDidMount() {
-    this.loadData();
+    const { history, location } = this.props;
```

Comment:
It's not the perfect location, but it works. 
I'd like to sync react-router with redux , like [this one](https://github.com/supasate/connected-react-router).